src/main/java/api/utils/CategoryUtils.java

```diff
@@ -4,10 +4,10 @@
 
 class CategoryUtils {
 
-    static Category generateDefaultCategory() {
+    static Category generateDefaultCategory(int id, String name) {
```

Comment:
here you could add default id and name, e.g.: 

int id = RandomUtils.nextInt(1, 5), String name = "RandomStringUtils.randomAlphabetic(20)"

you could then call this method with default values without specifying params like: generateDefaultCategory()
or in test you overriding the values with any input you want: generateDefaultCategory(22, "Something")

Reply:
Updated with the suggestion.